### PR TITLE
New resource tests

### PR DESCRIPTION
**Pull request by zbuchheit**

### CODE REVIEW

File: .github/workflows/build_sdk.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ env:
   MIST_API_TOKEN: ${{ secrets.MIST_API_TOKEN}}
   MIST_HOST: ${{ secrets.MIST_HOST}}
   MIST_ORG_ID: ${{ secrets.MIST_ORG_ID}}
+  MIST_CLAIM_CODE_1: ${{ secrets.MIST_CLAIM_CODE_1}}
+  MIST_CLAIM_CODE_2: ${{ secrets.MIST_CLAIM_CODE_2}}
```
Comment (blampe) [inline .github/workflows/build_sdk.yml:16]: These workflows are managed by ci-mgmt (https://github.com/pulumi/ci-mgmt/pull/1050) and will be overwritten. This is fine if you need it to temporarily unblock something, but ideally you'd configure these env vars in ci-mgmt.

Comment (VenelinMartinov) [reply .github/workflows/build_sdk.yml:16]: We have a provider-local config for env vars: https://github.com/pulumi/pulumi-gcp/blob/0c9738972c4c9a8bd444ee575311f8c4bd38aeaf/.ci-mgmt.yaml#L12

That should work here, too AFAIK - you need to add the variables to .ci-mgmt.yaml in the repo and then run `make ci-mgmt` @zbuchheit 

LMK if you need help

Comment (zbuchheit) [reply .github/workflows/build_sdk.yml:16]: is it necessary to run make ci-mgmt afterwards? It just re-orders the variables once I added them.

Comment (VenelinMartinov) [reply .github/workflows/build_sdk.yml:16]: Well, it'd get run overnight anyway, so it's better to run it yourself to make sure the changes are correct.

**Review by VenelinMartinov — APPROVED** [review]

File: .github/workflows/build_sdk.yml (hunk 1 above)
Comment (zbuchheit) [reply .github/workflows/build_sdk.yml:14]: went ahead and ran it. Thanks! The only thing that looked slightly odd is `--repository="github://api.github.com/pulumi"` being added to publish.yml but if that is expected by you all, then good with me.

Comment (VenelinMartinov) [reply .github/workflows/build_sdk.yml:14]: Brought it up in slack, it's not related to your changes, so you don't have to worry about it.

@danielrbradley confirmed it should be fine

**Pull request merged**